app/models/event.rb

```diff
@@ -17,9 +18,9 @@ def low_income_open?
     if are_missing
       false
     else
-      inside_window = !Time.zone.now.before?(low_income_requests_start) &&
-                      !Time.zone.now.after?(low_income_requests_end)
-      inside_window && event_mode == 'prerelease'
+      outside_window = Time.zone.now.before?(low_income_requests_start) ||
+                       Time.zone.now.after?(low_income_requests_end)
+      !outside_window && event_mode == 'prerelease'
```

Comment:
This is clearer, but I was thinking:
```
inside_window? = Time.zone.now.after?(low_income_requests_start) && Time.zone.now.before?(low_income_requests_end)
inside_window? && event_mode == 'prerelease'
```

that way it removes any negatives, which makes the logic easier to follow I think

Reply:
We can't have the question marks against `inside_window` but would be happy to remove the negatives for this check